SimPEG/EM/NSEM/Utils/dataUtils.py

```diff
@@ -54,7 +54,114 @@ def rotateData(NSEMdata, rotAngle):
 
     return Data.fromRecArray(outRec)
 
+# Function to get data and data info
+def extract_data_info(NSEMdata):
```

Comment:
thoughts on naming here? I what do you mean by `info` (survey details?) perhaps
```
extract_survey_info
```
? or something along those lines